index.html

```diff
@@ -350,7 +348,216 @@
+                        &lt;div class="o-grid  u-alignCenter  u-between  u-justifyCenter@l"&gt;
+                            &lt;div class="o-grid__item  u-size-6  u-size-2@m"&gt;
+                            &lt;div class="o-grid__item  u-size-4@m"&gt;
+                            &lt;div class="o-grid__item  u-size-6  u-size-2@m"&gt;
+                        &lt;/div&gt;
+                        </code>
+                    </pre>
+
+                    <p>Head over to the utilities section to see the full range of helpers on offer and how you can use them in your project.</p>
+
+                    <div class="u-h6  u-mT-32" id="usingoffsetutilities">Using offset utilities</div>
+
+                    <hr />
+
+                    <p>Sometimes if your designer is feeling particularly edgy they may create a wonderful arymetric page layout with areas that are being pulled and pushed outside the confines of the grid.. and you need to somehow create this using a grid system 😳</p>
+
+                    <p>This is where the offset utility can help you out!</p>
+
+                    <pre class="line-numbers">
+                        <code class="language-markup">
+                            &lt;div class="o-grid"&gt;
+                                &lt;div class="o-grid__item  u-size-8@l u-outsetLeft-2@l"&gt;&lt;/div&gt;
+                                &lt;div class="o-grid__item  u-size-6@l"&gt;&lt;/div&gt;
+                            &lt;/div&gt;
+
+                            &lt;div class="o-grid"&gt;
+                                &lt;div class="o-grid__item  u-size-4@l  u-inset-2@l"&gt;&lt;/div&gt;
+                                &lt;div class="o-grid__item  u-size-7@l  u-outsetRight-1@l"&gt;&lt;/div&gt;
+                            &lt;/div&gt;
+                        </code>
+                    </pre>
+
+                    <div class="u-h6  u-mT-32" id="includingthemixin">Including the mixin</div>
+
+                    <hr />
+
+                    <p>Usage of the grid mixin is simple, you can import it like so:</p>
+
+                    <pre class="line-numbers">
+                        <code class="language-sass">
+                            @include nu-object__grid();
+                        </code>
+                    </pre>
+
+                    <p>Or you can go anonymous and import the guts of the object without the parent class name, this is super useful when you want to make serious changes to a Neue Kit mixin.</p>
+
+                    <pre class="line-numbers">
+                        <code class="language-sass">
+                            .mySuperGrid {
+                                @include _nu-object__grid();
+                            }
+                        </code>
+                    </pre>
+
+                    <div class="u-h6  u-mT-32  u-mB-16" id="configuringthemixin">Configuring the mixin</div>
+
+                    <table class="table">
```

Comment:
Are we missing table element styles? Perhaps my build is wrong somehow?

![screenshot 2019-02-28 at 10 03 16](https://user-images.githubusercontent.com/8170676/53558543-1bb05f00-3b40-11e9-81bc-862db0f2a60d.png)

Reply:
Yea we need some table styles, we spoke about it some time ago in regards to making them responsive